build/generate-readme-api-tables/generate-api-tables.js

```diff
@@ -84,22 +84,37 @@ function getPropsTable({ displayName, props, tags }, tablePrefix) {
 	const hasThemableProps = props && Object.values(props).find((propInfo) => propInfo.themable);
 	const themeKey = displayNameToThemeKey(displayName);
 	const description = hasThemableProps ? `Themable props* can be configured via the [Theme](#/Theme) component using the key ${code(themeKey)}.` : undefined;
-
+	const standardProps = props ? Object.fromEntries(Object.entries(props).filter((prop) => !getPropDescription(prop[1]).includes('Advanced: '))) : undefined;
+	const advancedPropsArray = props ? Object.entries(props).filter((prop) => getPropDescription(prop[1]).includes('Advanced: ')) : undefined;
+	const advancedProps = advancedPropsArray && advancedPropsArray.length > 0
+		? Object.fromEntries(advancedPropsArray) : undefined;
 	return section(
 		`${tablePrefix ? `${tablePrefix} ` : ''}Props`,
 		[
 			inheritsFrom,
 			description,
-			props ? markdownTable([
+			standardProps ? markdownTable([
 				['Prop', 'Type', 'Default', 'Possible values', 'Description'],
-				...mapIn(props, (propInfo) => [
+				...mapIn(standardProps, (propInfo) => [
 					getPropName(propInfo),
 					getPropType(propInfo),
 					getPropDefaultValue(propInfo),
 					getPropPossibleValues(propInfo),
 					getPropDescription(propInfo),
 				]),
 			]) : '',
+			advancedProps ? '### Advanced customization props' : '',
```

Comment:
For the advancedProps title can we do this instead?:
```
`${tablePrefix ? `${tablePrefix} ` : ''}Props (Advanced)`,
```
I think it matches the title scheme a bit better, also will work for directories which export multiple components, e.g. ActionBar, which exports ActionBarButton and InlineActionBar both with their own sets of props